recorder.go

```diff
@@ -74,24 +96,35 @@ func getHostName(rawSpan basictracer.RawSpan) string {
 }
 
 func getServiceName(rawSpan basictracer.RawSpan) string {
-	s := getStringTag(rawSpan, string(ext.Component))
-	if s == "" {
-		s = getStringTag(rawSpan, string(ext.PeerService))
-		if s == "" {
-			return sensor.serviceName
+	// ServiceName can be determined from multiple sources and has
+	// the following priority (preferred first):
+	//   1. If added to the span via the OT component tag
+	//   2. If added to the span via the OT http.url tag
+	//   3. Specified in the tracer instantiation via Service option
```

Comment:
@boyand It's a good point.  Should the tracer setting override the span tags or vice versa?  I'm not sure but it probably doesn't matter for now until the new service mapping support is out.  I'll add a note and revisit.